.github/actions/reproducibility_index/action.yml

```diff
@@ -0,0 +1,108 @@
+name: Build Reproducibility Index
+description: 'Updates the reproducibility index for the given path'
+inputs:
+  binary_path:
```

Comment:
Is there a convention for these names? We have two here and they are in different capitalizations already